Jenkinsfile

```diff
@@ -1224,6 +1224,10 @@ pipeline {
                                   if [ -n "$STAGE_NAME" ]; then
                                       rm -rf "$STAGE_NAME/"
                                       mkdir "$STAGE_NAME/"
+                                      # compress those potentially huge DAOS logs
+                                      if daos_logs=$(ls src/tests/ftest/avocado/job-results/*/daos_logs/*); then
```

Comment:
Should we be a little more selective?   It's easier to view logs in the browser so perhaps we should reserve compression for ones that are larger?

Reply:
Interesting.  They never open in the browser for me, even uncompressed.

So we certainly could be size selective.  What kind of size threshold are you thinking of?  Are most useful logs below that threshold or would you end up only using the compressed logs anyway because the ones that are below the threshold don't really have anything useful in them anyway?

The other issue is that I've already spent two days trying to get this through test and having to manually go compress logs while waiting for it to get through test so that it can land.

Reply:
If click the "view" bottom of the log file instead of clicking the filename, it can be opened in browser, I would think <=32MB is the size for opening in browser, I wouldn't try to open things larger than that in my browser. Could you take a follow-on action to make it be size selective and only compress those things larger than 32MB (or 16MB if you worry about space consumption)

On the other hand, I think we should change the log bitmask for some D_DEBUG which are not very helpful, I can take care of this.